docs/specs/clients/notify/client-sdk-api.md

```diff
@@ -56,17 +58,32 @@ abstract class Client {
     encryptedMessage: string,
   }): Promise<NotifyMessage>;
   
+  // Generates and stores an identity key, indexed via the following key
+  // account-domain-isLimited
+  // Generates and returns SIWE message for said identity key
+  public abstract getSiweMessage(params: {
+    account: string;
+    domain: string,
+    isLimited?: boolean,
```

Comment:
Can we take this opportunity to change this to `allApps?: boolean = false` rather than `isLimited?: boolean = true`. It's more confusing as-is and the statements and links all talk about this app or all apps, not limited vs unlimited. Docs would be a lot easier to understand if this was changed. I had added this to my feature requests here: https://github.com/WalletConnect/web3inbox-widget/issues/40#issuecomment-1821717867